btk/draw_blends.py

```diff
@@ -249,10 +249,12 @@ def __next__(self):
             wcss[s.name] = wcs
 
             input_args = []
+            ss = np.random.SeedSequence(self.seed)
+            seeds_minibatch = ss.spawn(self.batch_size // mini_batch_size + 1)
```

Comment:
If I get it right, each time `__next__()` is called, the seed sequence is re-initialised with the same `self.seed`. 
Hence the minibatch seeds are identical every single time.

Is that what you want here ? (genuine question)

Reply:
Hmm, you're right, I'm not 100% sure that this is what we want. I think it would be better to initialize the `SeedSequence` in the `__init__`. Otherwise we will just get the same noise realization in each batch.